src/common/components/MultiToggleSwitch/index.ts

```diff
@@ -0,0 +1,3 @@
+import MultiToggleSwitch from './MultiToggleSwitch'
+
```

Comment:
@ArthurVogelsang Types should be exported from index.
`export type { TypeName } from './MultiToggleSwitch'`